client/src/components/TagSelector/Selection.tsx

```diff
@@ -0,0 +1,36 @@
+import type { SelectionProps } from "@/components/TagSelector/tag-selector.types";
+import { makePath } from "@/lib/tag-path";
+import type { TagWithIds } from "@/types/server/tag.types";
+import { Fragment } from "react/jsx-runtime";
+import S from "./TagSelector.style";
+
+function Selection(p: SelectionProps) {
+	if (!p.selectedTags.length) return null;
+
+	function renderPath(tag: TagWithIds, tags: TagWithIds[]) {
+		return (
+			<S.SelectionItem key={tag.tag_id}>
+				{p.fullPaths
+					? makePath(tag, tags)
+							.map((path, index) => (
+								<Fragment key={index}>
+									<S.PathPart $isLeaf={index === 0} key={path}>
```

Comment:
We do `index === 0`, and a few lines later `index !== 0`. Is it not better to put that in a variable (`isLeaf`) so that they always remain coupled and we can never forget to alter one line without the other?